lambda/passUtils.js

```diff
@@ -95,7 +95,7 @@ async function isBookingAllowed(orcs, facilitySk, date, type) {
 
     // the hour of day the next future day opens for booking (AM slot)
     let openingHour = facility.bookingOpeningHour || DEFAULT_AM_OPENING_HOUR;
-    let closingHour = DEFAULT_PM_OPENING_HOUR;
+    let closingHour = DEFAULT_PM_OPENING_HOUR
```

Comment:
`;` needed here.

Reply:
Fixed in: https://github.com/bcgov/parks-reso-api/pull/332/commits/2fe8069dc3d24f690e8fb6453218781c3c7c4133